src/alias/azext_alias/__init__.py

```diff
@@ -41,10 +50,17 @@ def __init__(self, cli_ctx=None):
         self.cli_ctx.register_event(EVENT_INTERACTIVE_POST_SUB_TREE_CREATE, enable_aliases_autocomplete_interactive)
 
     def load_command_table(self, _):
+
+        def remove_all_confirmation(_):
+            return bool(prompt_y_n('Are you sure you want to remove all registered aliases?'))
```

Comment:
It returns True/False already right?
https://github.com/Microsoft/knack/blob/master/knack/prompting.py#L80-L96

Reply:
Also, you don't need this. You can just pass the string as `confirmation='Are you sure you want to remove all registered aliases?'` right?

Reply:
https://github.com/Microsoft/knack/blob/master/knack/commands.py#L117